src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -152,135 +152,262 @@ internal partial class CultureData
         /// </remarks>
         private static Dictionary<string, string> RegionNames =>
             s_regionNames ??=
-            new Dictionary<string, string>(211 /* prime */)
+            new Dictionary<string, string>(257 /* prime */)
```

Comment:
nit, I don't think there is any value in passing in a prime number - the dictionary internally rounds up to the next prime number from its list. it's harmless though of course.

Ideally the C# compiler would know that there's N entries coming in the initializer and pass in N to the constructor.

Reply:
I'll keep it as it is harmless and I am not sure what the C# initializer how it behaves there.

Reply:
fine, fyi the initializer always uses the parameterless constructor, so it is a valid microoptmization to pass a size here.